src/sas/sascalc/invariant/invariant.py

```diff
@@ -849,7 +861,7 @@ def get_surface(self, contrast, porod_const, extrapolation=None):
 
     def get_volume_fraction(self, contrast, extrapolation=None):
         """
-        Compute volume fraction is deduced as follow: ::
+        Compute volume fraction is deduced as follows: ::
```

Comment:
I think if you say `... follows::` then it will produce "... follows:" in markup followed by a literal block.

Similarly below.

Reply:
Yes indeed; I just left it in the style of the document which seemed to use ": ::" throughout.